be/src/vec/data_types/data_type_factory.cpp

```diff
@@ -20,6 +20,10 @@
 
 #include "vec/data_types/data_type_factory.hpp"
 
+#include "vec/data_types/data_type_hll.h"
+#include "vec/data_types/data_type_jsonb.h"
```

Comment:
hll and jsonb already included in data_type_factory.hpp

Reply:
done